src/epics/router.ts

```diff
@@ -130,12 +133,47 @@ const changeView: Epic = (action$, store) => action$.ofType(types.selectView)
 
 const drillDown: Epic = (action$, store) => action$.ofType(types.drillDown)
 .switchMap(action => {
-    // const state = store.getState() as Store
+    const state = store.getState() as Store
```

Comment:
`as Store` is not required anymore as `store.getState` return value now has a correct typing

Reply:
Fixed